src/main/java/com/redhat/devtools/intellij/tektoncd/actions/StartAction.java

```diff
@@ -60,18 +62,32 @@ public void actionPerformed(AnActionEvent anActionEvent, TreePath path, Object s
                 logger.error("Error: " + e.getLocalizedMessage());
                 return;
             }
-            String finalConfiguration = configuration;
-            StartDialog stdialog = UIHelper.executeInUI(() -> {
-                StartDialog dialog = new StartDialog(null, finalConfiguration, resources);
-                dialog.show();
-                return dialog;
-            });
-            if (stdialog.isOK()) {
+
+            StartResourceModel model = new StartResourceModel(configuration, resources);
+
+            if (!model.isValid()) {
+                UIHelper.executeInUI(() -> Messages.showErrorDialog(model.getErrorMessage(), "Error"));
+                return;
+            }
+
+            boolean noInputsAndOuputs = model.getInputs() == null && model.getOutputs() == null;
```

Comment:
Just for clarity I think it's better if inputs and outputs are always there (never null) but check should be isEmpty

Reply:
postponed to #53